stable/kubernetes-dashboard/values.yaml

```diff
@@ -128,6 +128,11 @@ rbac:
   # ServiceAccount (NOT RECOMMENDED).
   clusterAdminRole: false
 
+  # Start in ReadOnly mode
+  # Only dashboard-related Secrets and ConfigMaps will still be available for writing.
+  # Turn OFF clusterAdminRole to use clusterReadOnlyRole
```

Comment:
Here. We need something explaining the ramifications of this... readonly sounds pretty benign. But this feature would allow anyone to be able to:
view pod logs. incorrectly logged entries can be gathered. This often includes logs of users accidentally typing passwords into user fields.
view pv contact info. This info could be used to get into a volume, or at minimum discover a previously unknown storage backend.

Reply:
Added more information to these lines, sqashed with the first commit. Thank you for reviewing.